File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisParameterNotFitException.java
package org.irislang.jiris.core.exceptions.fatal;

/**
 * Created by Huisama on 2017/5/27 0027.
 */
public class IrisParameterNotFitException extends IrisFatalException {
    public IrisParameterNotFitException(String fileName, int lineNumber, String message) {
        super(fileName, lineNumber, message);
    }

    @Override
    public String GetFatalExceptionName() {
        return "ParameterNotFitIrregular";
    }
}

File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisParameterNotFitException.kt
package org.irislang.jiris.core.exceptions.fatal

/**
 * Created by Huisama on 2017/5/27 0027.
 */
class IrisParameterNotFitException(fileName: String, lineNumber: Int, message: String) : IrisFatalException(fileName, lineNumber, message) {

    override fun GetFatalExceptionName(): String {
        return "ParameterNotFitIrregular"
    }
}
